cache/option.go

```diff
@@ -31,3 +32,10 @@ func WithFallbackGenerator(g key.Generator) Option {
 		o.fallbackGenerator = g
 	})
 }
+
+// WithOverride sets object should be overriten even if it already exists.
+func WithOverride() Option {
```

Comment:
This should take the value and set it. 
If nothing, it gives the ability to change behaviour when we test it.